### PR TITLE
Feature/document

### PR DESCRIPTION
**Pull request by hammad-Ikhlaq-7vals**
Document Added in issues.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_09_083144) do
+ActiveRecord::Schema.define(version: 2020_09_18_092018) do
```
Comment (mujtaba-saboor) [inline db/schema.rb:13]: why is this being pushed

File: app/controllers/issues_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class IssuesController < ApplicationController
+
```
Comment (mujtaba-saboor) [inline app/controllers/issues_controller.rb]: @UsmanAhmad169 's issue-PR once gets merged into develop, reverse merge this branch in peer-programming with him so it is done quickly. @hammad-Ikhlaq-7vals

**Comment by mujtaba-saboor** [comment]
cycle 1

File: db/schema.rb (hunk 1 above)
Comment (hammad-Ikhlaq-7vals) [reply db/schema.rb:13]: It is already in .gitignore but still it appears.

File: app/controllers/issues_controller.rb (hunk 2 above)
Comment (hammad-Ikhlaq-7vals) [reply app/controllers/issues_controller.rb]: ok

**Comment by mujtaba-saboor** [comment]
will be merged once other PRs get merged

**Pull request merged**